client/my-sites/checkout/src/components/additional-terms-of-service-in-cart.tsx

```diff
@@ -381,6 +382,32 @@ function getMessageForTermsOfServiceRecordUnknown(
 		},
 	};
 
+	if (
+		args.product_meta &&
+		args.product_meta !== '' &&
+		args.domain_transfer_slug === domainProductSlugs.TRANSFER_IN
+	) {
+		return translate(
+			'The promotional period of your %(productName)s for %(domainName)s will begin once the {{domainTransferSupportLink}}domain transfer is completed{{/domainTransferSupportLink}}. At that time, you will be notified of the promotional period, renewal date, and renewal price (%(renewalPrice)s) via email.',
```

Comment:
But we don't notify them via email of this yet (at least not in a meaningful way)..... I'm also not sure if this is going to be the best way to phrase it in the end.

For now, at least, we should probably put this on hold until more work is done on the email (in https://github.com/Automattic/wp-calypso/issues/79752 or elsewhere)?  I'm going to do that in the Shilling board (for now).

Perhaps I'll also finally have time to pick up what I was doing on https://github.com/Automattic/payments-shilling/issues/1850 again and phrase it in a different way, though.

Reply:
@DavidRothstein Would it be better to add this type of TOS after https://github.com/Automattic/payments-shilling/issues/2491 has been completed? 

That way, the receipt email for the domain registration product will already be fixed so it shows the promotional information and pricing. Then, we can introduce this TOS change explaining to the user that the information will be available once the transfer completes.

Reply:
Yeah, looks like it -- if that task were completed first, then we could safely mention it here.